draft-wang-ppm-differential-privacy.md

```diff
@@ -884,65 +910,129 @@ in order to achieve various combinations of `(EPSILON, DELTA)`-DP.
 
 ### Prio3Histogram with Aggregator-DP
 
-> TODO Describe a policy that is compatible with Prio3Histogram and that
-> targets the OAOC trust model.
+Aggregator-DP requires Aggregators to add noise to their aggregate shares before
+outputting them. Under OAOC trust model, we can achieve good `EPSILON`-DP, or
+`(EPSILON, DELTA)`-DP, as long as at least one of the Aggregators is honest. The
+amount of noise needed by Aggregator-DP typically depends on the target DP
+parameters `EPSILON`, `DELTA`, and also the `SENSITIVITY` of the aggregation
+function.
+
+In this policy, we will describe how to achieve `(EPSILON, DELTA)`-DP, by asking
+each Aggregator to independently add discrete Gaussian noise to its aggregate
+share, and by using `Prio3Histogram` VDAF for aggregation and robustness
+considerations.
+
+#### Discrete Gaussian Mechanism for Aggregator-DP
+
+The Aggregator-DP we will use here is the discrete Gaussian mechanism described
+in {{discrete-gaussian}}, which has mean of 0, and is initialized with a `SIGMA`
+parameter that stands for the standard deviation of the Gaussian distribution.
+In order to achieve `(EPSILON, DELTA)`-DP under OAOC trust model, all
+Aggregators need to independently add discrete Gaussian noise to all coordinates
+of their aggregate shares.
+
+To compute the parameter `SIGMA` for discrete Gaussian, Theorem 7 in {{CKS20}}
+and Theorem 8 in {{BW18}} have shown that the `SIGMA` parameter can be tuned to
+achieve `(EPSILON, DELTA)`-DP, for a given L2-`SENSITIVITY` {{sensitivity}} of
+an aggregation function. In a histogram use case where each Client submits an
```

Comment:
"To compute the parameter ... an aggregation function.": This sentence can be simplified. How about something like: "Theorem 7 in {{CKS20}} shows how to compute `SIGMA` for given approximate-DP parameters and L2-sensitivity."

Reply:
Good point. Fixed.